move-vm-backend/tests/assets/move-projects/substrate_balance/sources/Transfer.move

```diff
@@ -0,0 +1,31 @@
+script {
+    use substrate::balance;
+    use std::string;
+
+    fun balance_simple_api_test(src: signer, dst: address, amount: u128) {
+        // Test that the move-stdlib works fine.
```

Comment:
Is this test essential for `balance_simple_api_test`?
Is `string` used for further tests or what's the aim to check if `move-stdlib` works fine inside this test?

Reply:
I wrote this test to ensure we can publish two different unrelated bundles under the 0x1 address.
To really test it, I tried to use modules from those two different bundles - so that's the reason behind.

Maybe I can leave a comment about this. Maybe this is obvious to expert Move devs, but I wanted to check it with this test.

Reply:
Done